src/index.ts

```diff
@@ -41,6 +41,16 @@ app.post("/users", async (req, res) => {
   res.send({ user });
 });
 
+app.use((req, res): void => {
+  res.status(404)
+    .send({ message: "Invalid Route"})
+})
+
+app.use((error: Error, req: Request, res: Response, next: NextFunction):void => {
+  res.status(500)
+    .send({ message: "Oops! Server Error" })
```

Comment:
Not necessary for now, but in the future we may want to craft clear error messages for each error case and send that to the front-end instead